LoopCore/Insulin/ExponentialInsulinModelPreset.swift

```diff
@@ -38,9 +38,20 @@ extension ExponentialInsulinModelPreset {
             return .minutes(55)
         }
     }
+    
+    var effectDelay: TimeInterval {
```

Comment:
@dm61 ideally, this should be user configurable (due to YDMV) IMHO. In many cases 15 or even 20 minutes might be more adequate (without the user needing to make a code change).

Reply:
Thanks for the comment. The main purpose of this PR is to address a slight inconsistency in dosing versus prediction algorithms, as discussed in  https://github.com/LoopKit/Loop/pull/1238. UI-related decisions and design are separate issues, beyond the scope of this particular PR.